flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/HDFSSequenceFile.java

```diff
@@ -98,11 +95,11 @@ protected void open(Path dstPath, CompressionCodec codeC,
       if(hdfs instanceof LocalFileSystem) {
         hdfs = ((LocalFileSystem)hdfs).getRaw();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "hdfs". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1226)